### PR TITLE
example: add example for parsing kubernetes logs

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

**Description:** <Describe what has changed.>

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by pmalek-sumo** [comment]
in commit title `exmaple` -> `example`

and please resolve conflicts.

File: examples/kubernetes/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: "3"
+services:
+  opentelemetry-collector-contrib:
+    image: otelcontribcol
+    command: ["--config=/etc/otel-collector-config.yml"]
+    volumes:
+      - ./otel-collector-config.yml:/etc/otel-collector-config.yml
+      - ./varlogpods:/var/log/pods
+
```
Comment (sumo-drosiek) [inline examples/kubernetes/docker-compose.yml:9]:
```suggestion
 
```

File: examples/kubernetes/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
```
Comment (sumo-drosiek) [inline examples/kubernetes/docker-compose.yml:9]:
```suggestion

```

**Comment by sumo-drosiek** [comment]
@pmalek-sumo please check again

File: examples/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# OpenTelemetry Collector Demo
+
+This demo is a sample app to build the collector and exercise its kubernetes logs scrapping functionality
```
Comment (pmalek-sumo) [inline examples/kubernetes/README.md:3]:
```suggestion
This demo is a sample app to build the collector and exercise its kubernetes logs scrapping functionality.
```

File: examples/kubernetes/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+`varlogpods` contains example log files placed in kubernetes-like directory structure.
+Each of the directory has different formatted logs in one of three formats (either `CRI-O`, `CRI-Containerd` or `Docker`).
+This directory is mounted to standard location (`/var/log/pods`).
+
+`otel-collector-config` is a configuration to autodetect and parse logs for all of three mentioned formats
+
+## ToDo
+
+To cover kubernetes system logs, logs from journald should be supported as well
```
Comment (pmalek-sumo) [inline examples/kubernetes/README.md:22]:
```suggestion
To cover kubernetes system logs, logs from journald should be supported as well.
```

**Review by pmalek-sumo — APPROVED** [review]

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**